src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -95,6 +104,10 @@ public GeoApiContext() {
       }
     }
 
+    if(channelSet==false && channel != null && !channel.isEmpty() ){
+      query.append("&chanel=").append(channel);
```

Comment:
Typo here - should be "&channel=". 

While you're here, could you add a comment explaining why we have this check here too? e.g. // Channel can be supplied per-request or per-context. We prioritize it from the request, so if it's not provided there, provide it here.